core/sessions/orm.go

```diff
@@ -234,6 +241,13 @@ func (o *orm) CreateSession(sr SessionRequest) (string, error) {
 		return "", err
 	}
 
+	// Forward registered credentials for audit logs
+	uwasj, err := json.Marshal(uwas)
+	if err != nil {
+		lggr.Errorf("error in Marshal credentials: %s", err)
+	}
+	o.auditLogger.Audit(audit.AuthLoginSuccessWith2FA, map[string]interface{}{"email": sr.Email, "credential": string(uwasj)})
```

Comment:
Should/must this be skipped if we fail to marshal?
```suggestion
	} else {
		o.auditLogger.Audit(audit.AuthLoginSuccessWith2FA, map[string]interface{}{"email": sr.Email, "credential": string(uwasj)})
	}
```